All EIPIP Meetings/Meeting 011.md

```diff
@@ -0,0 +1,552 @@
+
+**James Hancock**
+Yeah, I would agree.
+
+**Pooja Ranjan**
+I'm not too sure how much value it would add if it is not something that is going to be commented. And in most of the EIPs, we can probably go ahead and explain that term in one of the summary sections or somewhere else in the EIP itself. But adding this as a mandatory section for every EIP that is something I'm not sure about, but it doesn't hurt to have something additional as well.
+
+**Edson Ayllon**
+So I think it's good as an optional section. I don't think that it needs to be mandatory for every single EIP.
+
+**James Hancock**
+Yeah, it would be good to have one in some of them going to EIP 1. We can have definitions that people can use, like the definition of a network upgrade.
+
+**Pooja Ranjan**
+Right, that is that is needed for EIP one. I understand this point. This section is mandatory for EIP 1. But again, EIP 1 is not any general EIP. It's a kind of documentation that provides a guideline. So for that, having this section, it makes a lot of sense. But making it mandatory, I was not sure. But if it is optional, yeah, perfectly fine.
+
+**James Hancock**
+When I've written EIPs, I tried to do this or be mindful of if I'm using terms defining them. So having that be as a best practice, is something else we should consider. Because we have stuff that's required. And then we're starting to also list things as best practices. And that, definitely, we should add to the list of best practices. Or consider doing it. I don't want to decide. I'm getting ahead of myself. We're at the top of the hour. So unless there's some final thoughts, I think it's time to wrap up.
+
+**Pooja Ranjan**
+Can we just quickly summarize for the note takers, what are the decision makers and action items.
+
+**James Hancock**
+Decision and action items:
+
+1. Look at the Cat Herders to investigate the project board for the EIP repo. I mean the EIP ip repo
+2. Follow up with lightclient on his new validator and the rules.
+3. James to draft PR for new statuses
+
+**Pooja Ranjan**
+Did we decide on any timeline to submit this PR or is it final or we would want to wait? And what do you think about it? The PR for the post mortem update I'm talking about?
+
+**James Hancock**
+I don't think we need to decide a timeline. We should meet again and keep talking about it. Then bring it up on the next call.
+
+**Pooja Ranjan**
+Okay.
+
+**James Hancock**
+And then the next step on that is to  present it on the AllCoreDevs call.
+
+**Pooja Ranjan**
+Oh, yeah, of course. I'm not sure for AllCoreDevs, should it be submitted as a PR in the EIP repository or it can be shared just like that?
+
+**James Hancock**
+Either. Yeah. Are there outstanding action items that I'm forgetting?
+
+**Pooja Ranjan**
+Yeah, and we would like to bring some of some of the agendas are repeated in the next meeting. We would like to continue the discussion on rest of the items.
```

Comment:
Yeah, and we would like to bring some of the agendas here repeated in the next meeting. We would like to continue the discussion on the rest of the items.